google/generativeai/responder.py

```diff
@@ -0,0 +1,453 @@
+from __future__ import annotations
```

Comment:
We don't have the a license check in the presubmits?

Please add the license here.

Reply:
Added license.